### DISCUSSION TITLE
Play all the loved tracks in the playlist

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/Tauon#733] [Ideas]
I have a new idea today! What if we could play only the loved tracks in the playlist? Or maybe in all playlists? Currently I can add all the loved tracks to the queue by middle click as a workaround.

**Comment by Taiko2k** [comment]
Just do <kbd>Ctrl</kbd> + <kbd>t</kbd> then <kbd>Ctrl</kbd> + <kbd>e</kbd> then type: `a r l`, <kbd>Enter</kbd>

**Reply by Solarunit** [reply]
_Is this some kind of secret combo to defeat a final boss?_ :laughing: 
I think you should add this to the documentation.

____

Ok, this is so cool! 
Now I have a playlist with all loved tracks! :tada: 
Any way to get a loved tracks from the one particular playlist maybe?

**Reply by Taiko2k** [reply]
Sure you can do it from a particular playlist. You can check the documentation [here](https://github.com/Taiko2k/TauonMusicBox/wiki/Generator-Codes). Or, for a single playlist, you can also do it the UI way by just right clicking the playlist tab > Genterate..., and there's an option for loved tracks there.

**Reply by Solarunit** [reply]
Oh, that was easy, thank you!
